.github/workflows/deploy.yml

```diff
@@ -24,8 +24,6 @@ jobs:
       - run: echo "::set-output name=version::$(git rev-parse origin/master)"
         id: pre-version
       - uses: bahmutov/npm-install@v1
-      - name: Delete old builds
```

Comment:
don't delete old builds, so that when deploying, we can continue running the old frontend in parallel with the new one